v2/pkg/engine/datasource/grpc_datasource/execution_plan_visitor.go

```diff
@@ -211,8 +211,6 @@ func (r *rpcPlanVisitor) EnterSelectionSet(ref int) {
 			}
 			resolvedField := &r.resolverFields[resolverFieldAncestor]
 			resolvedField.memberTypes = memberTypes
-			resolvedField.fieldsSelectionSetRef = ast.InvalidRef
```

Comment:
Technically this means `resolvedField.fieldsSelectionSetRef` is now 0 instad of -1 but doesn't this logically mean we declare this fields selection set reference as valid whereas before it was invalid? Was it a bug?